helix-rest/src/main/java/org/apache/helix/rest/server/service/InstanceService.java

```diff
@@ -49,6 +47,7 @@ InstanceInfo getInstanceInfo(String clusterId, String instanceName,
      * @return An instance of {@link StoppableCheck} easily convertible to JSON
      * @throws IOException in case of network failure
      */
+    @Deprecated
```

Comment:
Let's not deprecate them. They are still can be leveraged as module function just for checks.